snuba/utils/state_machine.py

```diff
@@ -55,14 +56,14 @@ def handle(self, state_data: TStateData) -> Tuple[TStateCompletionEvent, TStateD
 StateTransitions = Mapping[
     TStateCompletionEvent,
     Union[
-        StateType,
+        StateType[TStateCompletionEvent, TStateData],
```

Comment:
Is this why you were talking about having to respecify the type variables over and over the other day?

Reply:
yes